CellProfiler_pipelines/Metadata/barcode_platemap_Plate2.csv

```diff
@@ -0,0 +1,2 @@
+Assay_Plate_Barcode,Plate_Map_Name
+2,platemap_NF1_CP_Plate2
```

Comment:
red symbol 🙃

Reply:
Wait these happen on csvs too?! It is fixed :smile: